adsingestp/parsers/crossref.py

```diff
@@ -198,6 +198,10 @@ def _parse_contrib(self):
                 orcid = orcid.replace("http://orcid.org/", "")
                 contrib_tmp["orcid"] = orcid
 
+            if c.find("affiliation"):
+                affil = [a.get_text() for a in c.find_all("affiliation")]
```

Comment:
E.g. 10.3137/ao.410105 has an affiliation field, but it is empty (at least in the JSON data). I think we should skip empty affiliations.

Reply:
I added a check `if affil:` and only then to create `contrib_tmp["aff"]`.

Reply:
Looks good to me